File: Hash Maps/LinkedList.java
public class LinkedList{
  public Node head;

  public LinkedList(){
    this.head = null;
  }

  public void addToHead(String key, String value){
    Node newHead = new Node(key,value);
    Node currentHead = this.head;
    this.head = newHead;
    if(currentHead != null){
      this.head.setNextNode(currentHead);
    }
  }

  public void addToTail(String key, String value){
    Node tail = this.head;
    if(tail == null){
      this.head = new Node
    }
  }
    
}

File: Linkedlist/Single Linked List/linkedlist.py
class linkedlist:
    def __init__(self):
        self.head = None
        
    def addtohead(self,data):
        newhead = Node(data)
        current = self.head
        self.head = newhead
        if(current != None):
            self.head.getNodeNext(current)
    
    
    